libfreerdp/core/window.c

```diff
@@ -538,7 +657,7 @@ static BOOL update_recv_window_info_order(rdpUpdate* update, wStream* s,
 	else if (orderInfo->fieldFlags & WINDOW_ORDER_STATE_DELETED)
 	{
 		update_read_window_delete_order(s, orderInfo);
-		WLog_Print(update->log, WLOG_DEBUG, "WindowDelete");
+		WLog_Print(update->log, WLOG_DEBUG, "WindowDelete 0x%x", orderInfo->windowId);
```

Comment:
Please use the appropriate format string macros

Reply:
Ok